src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -138,6 +144,56 @@ public function getStyle()
         return $this->style;
     }
 
+    /**
+     * Sets table style.
+     *
+     * @param int               $columnIndex Column index
+     * @param TableStyle|string $name        The style name or a TableStyle instance
+     *
+     * @return Table
+     */
+    public function setColumnStyle($columnIndex, $name)
+    {
+        $isIndexValid = false;
```

Comment:
we should assume that  $columnIndex is an [int](https://github.com/MAXakaWIZARD/symfony/blob/pr-console-table-columns-styles/src/Symfony/Component/Console/Helper/Table.php#L150), IMO we can remove checking the [columnIndex](https://github.com/MAXakaWIZARD/symfony/blob/pr-console-table-columns-styles/src/Symfony/Component/Console/Helper/Table.php#L157-L169) type